bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx133-donation-eu-na.html

```diff
@@ -0,0 +1,120 @@
+  {% endif %}
+
+{% elif LANG == 'es-ES' %}
+  {% set main_body_part_one = "Firefox cuenta con el respaldo de la Fundación Mozilla, una comunidad global que trabaja para mantener un Internet abierto y accesible para todos. Depende de todos nosotros luchar contra las compañías tecnológicas irresponsables y proteger nuestra privacidad online." %}
+  {% set main_body_part_two = "Elegiste Firefox como tu navegador de Internet. Ahora te pedimos que des un paso más: haz una donación de fin de año a la Fundación Mozilla. Hoy mismo puedes colaborar a recuperar el control en la red." %}
+  {% set page_cta = "Donar a la Fundación Mozilla" %}
+
+  {% if variant == '2' %}
+    {% set main_title = "Haz una donación a la Fundación Mozilla antes de final de año" %}
+    {% set cta_link = "https://foundation.mozilla.org/?form=24-wnp-nov15" %}
+  {% else %}
+    {% set main_title = "Haz una donación a la Fundación Mozilla, la organización sin ánimo de lucro detrás de Firefox" %}
+    {% set cta_link = "https://foundation.mozilla.org/?form=24-wnp-nov16" %}
+  {% endif %}
+
+{% else %}
+  {% set main_body_part_one = "Firefox is backed by the Mozilla Foundation – a global community working to keep the internet open and accessible to all. It’s up to all of us to take on irresponsible tech companies and protect our privacy online." %}
+  {% set main_body_part_two = "You chose Firefox as your internet browser, and now we hope you’ll take the next step to reclaim the internet. Make an end-of-year donation to the Mozilla Foundation today." %}
+  {% set page_cta = "Donate to the Mozilla Foundation" %}
+
+  {% if variant == '2' %}
+    {% set main_title = "Donate to the Mozilla Foundation before our end-of-year deadline" %}
+    {% set cta_link = "https://foundation.mozilla.org/?form=24-wnp-nov17" %}
+  {% else %}
+    {% set main_title = "Donate to the Mozilla Foundation, the non-profit behind Firefox" %}
+    {% set cta_link = "https://foundation.mozilla.org/?form=24-wnp-nov18" %}
+  {% endif %}
+{% endif %}
+
+{% block wnp_content %}
+<section class="wnp-content-main">
+  {% call split(
+    image=resp_img('img/firefox/whatsnew/whatsnew133/mofo-heart.png',
+      srcset={ 'img/firefox/whatsnew/whatsnew133/mofo-heart-high-res.png': '2x' },
+      optional_attributes={ 'class': 'mzp-c-split-media-asset'}
+    ),
+    media_after=False,
+    media_class='mzp-l-split-v-center mzp-l-split-h-start',
+    block_class='mzp-l-split-reversed mzp-l-split-body-wide'
+    ) %}
+      <h1 class="wnp-main-title">{{ main_title }}</h1>
+      <p class="wnp-main-body">{{ main_body_part_one }}</p>
+      <p class="wnp-main-body"><strong>{{ main_body_part_two }}</strong></p>
+      <a class="wnp-main-cta mzp-c-button mzp-t-product" href="{{ cta_link }}" data-cta-text="Donate to the Mozilla Foundation">
+        {{ page_cta }}
+      </a>
+  {% endcall %}
+</section>
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle('firefox_whatsnew_133') }}
```

Comment:
**Suggestion:** You can remove this bundle and replace it with `firefox-whatsnew-events` (which has identical code)